tools/cli/adminFailoverCommands.go

```diff
@@ -273,6 +200,69 @@ func getRunID(c *cli.Context) string {
 	return ""
 }
 
+func failoverStart(c *cli.Context, params *startParams) {
+	validateStartParams(params)
+
+	targetCluster := params.targetCluster
+	sourceCluster := params.sourceCluster
+	batchFailoverSize := params.batchFailoverSize
+	batchFailoverWaitTimeInSeconds := params.batchFailoverWaitTimeInSeconds
+	workflowTimeout := time.Duration(params.failoverTimeout) * time.Second
+	domains := params.domains
+	drillWaitTime := time.Duration(params.drillWaitTime) * time.Second
+
+	client := getCadenceClient(c)
```

Comment:
I may not have the full context, looks like the failover workflow should run in a global system domain, if we have that?

Reply:
I think we decide to run in local domain. We can sync offline.